tests/integration/framework/process/daprd/daprd.go

```diff
@@ -76,9 +76,9 @@ func New(t *testing.T, fopts ...Option) *Daprd {
 		fopt(&opts)
 	}
 
-	dir := t.TempDir()
```

Comment:
Why change this? What is the `r` for?

Reply:
I see `resourceDirs` below, maybe the `r` is for resource? I would lean in favor of having more context then less on a var.